src/index.html

```diff
@@ -5,6 +5,7 @@
   <head>
     <link rel='stylesheet' type='text/css' href='scss/reset.css'>
     <link rel='stylesheet' type='text/css' href='scss/application.css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet'>
```

Comment:
it's better to add this in the scss file, just add
`@import 'https://fonts.googleapis.com/css?family=Open+Sans:400,700'`
at the begining of the scss file and that's it